torch/_dynamo/utils.py

```diff
@@ -3290,6 +3294,17 @@ def get_fake_value(node, tx, allow_non_graph_fake=False):
         _ = pytree.tree_map_only(
             torch.Tensor, functools.partial(ensure_graph_fake, tx=tx), ret_val
         )
+
+    if (
+        torch._dynamo.config.use_graph_deduplication
+        or torch._dynamo.config.track_nodes_for_deduplication
+    ):
+        tx.output.region_tracker.track_node_mutations(
```

Comment:
Why not track it in `output_graph`? Basically the tracking of node mutation is a separate thing that could be useful for other parts as well.

Reply:
or actually, I'll leave it, it requires a larger refactoring than I thought. I think they could move it if needed in the future.